src/frontend/state/authModalState.ts

```diff
@@ -0,0 +1,34 @@
+import { makeAutoObservable } from 'mobx'
+
+class AuthModalState {
+  private qaModeActive = false
+  private open = true
```

Comment:
we should use `window.api.handleQaModeActivated` to set open back to true. otherwise, I have to close and reopen HyperPlay to trigger the auth modal again